gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergPartitionDatasetFinder.java

```diff
@@ -38,6 +38,9 @@
 public class IcebergPartitionDatasetFinder extends IcebergDatasetFinder {
   public static final String ICEBERG_PARTITION_NAME_KEY = "partition.name";
   public static final String ICEBERG_PARTITION_VALUE_KEY = "partition.value";
+  public static final String ICEBERG_PARTITION_DATASET_PARTITION_SPEC_STRICT_EQUALITY = ICEBERG_DATASET_PREFIX + "strictEqualityForPartitionSpec";
```

Comment:
more common convention might be "validate.strict.partition.equality" or "strict.partition.equality.validation"

Reply:
Done